CONTRIBUTING.md

```diff
@@ -110,6 +110,29 @@ At this point, you should follow a standard git workflow:
 `make generate-in-docker` target which will install necessary tools in containerized environment and will generate assets.
 This targets needs `docker` to be installed on host and was not tested with other container runtime environments.
 
+## Testing
+
+Supposing $KUBECONFIOG is set to the config file of a Kubernetes cluster running the codes to test. We can run all tests by using this command `make test`.
+
+The testing consist of 3 aspects:
+- unit tests, can be run seperately by `make test-unit`.
+- Prometheus rule tests, can be run seperately by `make test-rules`.
+- end to end tests, can be run seperately by `make test-e2e`.
+Unit testing can be run by using `make test-unit` command.
```

Comment:
This phrase can be removed no? Since the first bullet point already tell us this
```suggestion
```